_data-prepper/pipelines/configuration/processors/geoip.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: default
+title: geoip
+parent: Processors
+grand_parent: Pipelines
+nav_order: 54
+---
+
+# geoip
+
+The `geoip` processor enriches events with geographic information from IP addresses contained inside the events.
+By default, Data Prepper uses the [MaxMind GeoLite2](https://dev.maxmind.com/geoip/geolite2-free-geolocation-data) geolocation database.
+Data Prepper administrators can configure the databases using the [`geoip_service`]({{site.url}}{{site.baseurl}}/data-prepper/managing-data-prepper/extensions/geoip_service) extension configuration.
+
+## Usage
+
+You configure the `geoip` processor to work on entries.
+
+The minimal configuration requires at least one entry and each entry needs a source field.
```

Comment:
Instead of "a source field", "one source field" or "at least one source field"?

Reply:
at least one source field.